src/gui/owncloudsetupwizard.cpp

```diff
@@ -346,7 +346,7 @@ void OwncloudSetupWizard::slotAuthError()
     }
 
     _ocWizard->show();
-    if (_ocWizard->currentId() == WizardCommon::Page_ShibbolethCreds) {
+    if (_ocWizard->currentId() == WizardCommon::Page_ShibbolethCreds || _ocWizard->currentId() == WizardCommon::Page_HttpCreds) {
```

Comment:
Oops, it was supposed to be the OAuth page of course.  Well spotted.

(This normally should not never happen because if we have a oauth connection, it should normaly work always. )